oauthproxy.go

```diff
@@ -70,6 +76,7 @@ type OAuthProxy struct {
 	AuthOnlyPath      string
 	UserInfoPath      string
 
+	allowedRoutes           []*allowedRoute
```

Comment:
Does this need to be to be a pointer or could it just be `[]allowedRoute`?

Reply:
I weighed the same thing when I was coding this 😄 

Switching it to non-pointer since you brought it up too.  The struct is pretty lightweight already, pointer doesn't buy us much.